hw/dv/doc/dv_plan_template.md

```diff
@@ -109,10 +109,10 @@ It creates the following analysis ports to retrieve the data monitored by corres
 ## Building and running tests
 We are using our in-house developed [regression tool]({{< relref "hw/dv/tools/README.md" >}}) for building and running our tests and regressions.
 Please take a look at the link for detailed information on the usage, capabilities, features and known issues.
-Here's how to run a basic sanity test:
+Here's how to run a basic smoke test:
```

Comment:
Nit: maybe this should just be "... run a smoke test"? I'm not sure what "basic" contributes here any more.

Reply:
Thanks Rupert, I will update that.